project-words.txt

```diff
@@ -679,4 +679,5 @@ redcircle
 bluecircle
 tatarot
 Tatarot
-Scry
\ No newline at end of file
+Scry
+ENSIP
```

Comment:
@arthur-remy 
can you do this in alphabetical order - there are merge conflicts

Reply:
Done